Can not build on Windows with "sm-raw-window-handle" feature

`cargo build --no-default-features --features "sm-raw-window-handle"`
results in:
```
error[E0432]: unresolved import `raw_window_handle::RawWindowHandle::Xlib`
   --> surfman\src\platform\windows\wgl\connection.rs:139:13
    |
139 |         use raw_window_handle::RawWindowHandle::Xlib;
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `Xlib` in `RawWindowHandle`

error[E0560]: struct `platform::windows::wgl::surface::NativeWidget` has no field named `window`
   --> surfman\src\platform\windows\wgl\connection.rs:143:17
    |
143 |                 window: handle.window,
    |                 ^^^^^^ `platform::windows::wgl::surface::NativeWidget` does not have this field
    |
    = note: available fields are: `window_handle`
```

--- a/surfman/src/platform/windows/wgl/connection.rs
+++ b/surfman/src/platform/windows/wgl/connection.rs
@@ -136,11 +136,11 @@ impl Connection {
         &self,
         raw_handle: raw_window_handle::RawWindowHandle,
     ) -> Result<NativeWidget, Error> {
-        use raw_window_handle::RawWindowHandle::Xlib;
+        use raw_window_handle::RawWindowHandle::Windows;
 
         match raw_handle {
-            Xlib(handle) => Ok(NativeWidget {
-                window: handle.window,
+            Windows(handle) => Ok(NativeWidget {
+                window_handle: handle.hwnd as HWND,
             }),
             _ => Err(Error::IncompatibleNativeWidget),
         }

